DB/projects.json

```diff
@@ -3555,5 +3555,22 @@
         "tech": ["html", "css"]
       }
     ]
+  },
+   {
+    "github_username": "utsavbhattarai007",
+    "Projects": [
+      {
+        "github_url": "https://github.com/utsavbhattarai007/devhub-with-appwrite",
+        "title": "Hackathon project",
```

Comment:
It would be helpful to users if you could provide a valid title for the project. This will make it easier for them to identify and understand what the project is about. Please consider adding a suitable title.